debian/ooniprobe-cli.service

```
@@ -0,0 +1,51 @@
+[Unit]
+Description=OONI CLI Probe
+Documentation=man:ooniprobe-cli
+#Documentation=file:///usr/share/doc/ooniprobe-cli/html/index.html
+Documentation=https://ooni.org/
+After=network.target tor.target
+Wants=network-online.target
+ConditionPathExists=/etc/ooniprobe/ooniprobe.conf
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/ooniprobe --config=/etc/ooniprobe/ooniprobe.conf run all
```

```suggestion
ExecStart=/usr/bin/ooniprobe --config=/etc/ooniprobe/ooniprobe.conf run unattended
```

We don't want to run performance tests in background. (The `unattended` command is implemented in 3.1.0)